test/Libraries/DynamoMSOfficeTests/ExcelTests.cs

```diff
@@ -7,6 +7,10 @@
 using DSOffice;
 using NUnit.Framework;
 using ProtoCore.Mirror;
+using Dynamo.DSEngine.CodeCompletion;
+using Dynamo.Models;
+using Dynamo.Nodes;
+using DynCmd = Dynamo.Models.DynamoModel;
```

Comment:
Could we rename this to `DynModel` or something? `DynCmd` gives the impression it refers to a command.